### ISSUE TITLE
Improve the usage of the quotes database

### ISSUE THREAD

**Issue by bcongdon** [acm-uiuc/groot-quotes-service#4]
Moderate the quotes so that they can be tied into other things. Think loading screens on Slack.

https://github.com/acm-uiuc/groot/issues/8

**Comment by bcongdon** [comment]
FWIW I kinda gave up on integrating Quotes with `groot-voz` because of how NSFW they are. This would be a good thing to work on.

**Comment by sameetandpotatoes** [comment]
Yeah, maybe quotes service can have a way to filter NSFW quotes (although it would have to be tested pretty thoroughly to make sure it doesn't let any NSFW quotes through)

**Comment by bcongdon** [comment]
Probably would be more maintainable to have a NSFW flag a la reddit
